BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -274,6 +286,55 @@ - (void)updateRequiredFields {
     }
 }
 
+#pragma mark - card.io
+
+- (void)setupCardIO {
+    if ([self isCardIOAvailable]) {
+        BTUIKInputAccessoryToolbar *cardToolBar = [BTUIKInputAccessoryToolbar new];
+        UIBarButtonItem *flexSpace = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil];        
+        UIBarButtonItem *cardIOButton = [[BTUIKBarButtonItem alloc] initWithTitle:BTUIKLocalizedString(SCAN_CARD_IO_ACTION) style:UIBarButtonItemStylePlain target:self action:@selector(presentCardIO)];
+        cardToolBar.items = @[flexSpace, cardIOButton, flexSpace];
+        self.cardNumberField.textField.inputAccessoryView = cardToolBar;
```

Comment:
I would look at using the `preloadCardIO` ( https://github.com/card-io/card.io-iOS-source/blob/master/CardIO_Public_API/CardIOUtilities.h#L49 ) method potentially if more people use card.io to improve the speed. Could be a future enhancement.